### ISSUE TITLE
Update docs-site.

### ISSUE THREAD

**Issue by AlexKVal** [react-bootstrap/react-bootstrap#971]
https://github.com/react-bootstrap/react-bootstrap/pull/947#issuecomment-119736859
I've tried and TIL I have no rights to do that.

--
Owners. Could you please do that ? I can help, but it is simple.

Steps needed to release docs:
```sh
> git clone git@github.com:react-bootstrap/react-bootstrap.git RB-master
> cd RB-master
> npm install
> ./tools/release —docs
```

`RB-master`  folder needed for those who works with own fork of R-B. Like this:
```sh
› git remote -v
origin	git@github.com:AlexKVal/react-bootstrap.git (fetch)
origin	git@github.com:AlexKVal/react-bootstrap.git (push)
upstream	https://github.com/react-bootstrap/react-bootstrap.git (fetch)
upstream	https://github.com/react-bootstrap/react-bootstrap.git (push)
```

That’s all. :sparkles:

**Comment by AlexKVal** [comment]
I thought that I have 'write' rights to commit to https://github.com/react-bootstrap/react-bootstrap.github.io
then I run `./tools/release —docs`. 
It did all well till the pushing to the site repo.
Because of that run we have now this in logs but without the site updated:
```sh
react-bootstrap/ on master
> git log
```
<img width="774" alt="screen shot 2015-07-11 at 10 44 21 am" src="https://cloud.githubusercontent.com/assets/847572/8632939/3711694e-27ba-11e5-8eae-d6f83e5812d4.png">

Sorry about that :cherries:

**Comment by taion** [comment]
Published `v0.23.7-docs.1`.
